internal/server/middleware.go

```diff
@@ -228,6 +228,8 @@ func requireAccessKey(c *gin.Context, db data.GormTxn, srv *Server) (access.Auth
 	}
 
 	// now that the org is loaded scope all db calls to that org
+	// TODO: set the orgID explicitly in the options passed to GetIdentity to
+	// remove the need for this NewTransaction call.
 	db = data.NewTransaction(db.GormDB(), org.ID)
```

Comment:
This is the last remaining call to `data.NewTransaction` that I'd like to remove (the other ones are in middleware or in create organization, and those are necessary).

To do this I think we need to replace selector funcs with a struct. Then we can easily validate that the struct includes a non-zero OrganizationID.  Technically we could do this today by inspecting the `db.Statement.Clauses["WHERE"]`, but that code is going to be very-gorm specific, so will be removed soon. For now it seemed easier to keep this todo. I'll see it again when we remove the selector func to the `data.GetIdentity` call below.